test/xbmcgui.py

```diff
@@ -191,6 +232,11 @@ def setInfo(type, infoLabels):  # pylint: disable=redefined-builtin
         ''' A stub implementation for the xbmcgui ListItem class setInfo() method '''
         return
 
+    @staticmethod
+    def setIsFolder(isFolder):
```

Comment:
So this one does not exist in Kodi Krypton. And for VRT NU we are testing also for Krypton.
Maybe we would need to add e.g. a decorator so functions or methods that are not supported would trigger and error.
And at the same time we need a mechanism to indicate the lowest version of Kodi that is supported.

```python
@leia
@staticmethod
def setIsFolder(isFolder):
```